sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -93,21 +98,23 @@ internal class OperationInternal<T> : OperationInternalBase
         /// <param name="scopeAttributes">The attributes to use during diagnostic scope creation.</param>
         /// <param name="fallbackStrategy">The delay strategy when Retry-After header is not present.  When it is present, the longer of the two delays will be used.
         ///     Default is <see cref="FixedDelayWithNoJitterStrategy"/>.</param>
+        /// <param name="requetMethod">The rehydration token.</param>
```

Comment:
The `requestMethod` comment seems wrong.   Please correct and format as a sentence to match existing docs.

Reply:
updated